frontend/src/metabase/user/components/UserSettings.jsx

```diff
@@ -12,13 +12,37 @@ import UserAvatar from "metabase/components/UserAvatar";
 import SetUserPassword from "./SetUserPassword";
 
 export default class UserSettings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { updateUserResult: null };
+  }
+
   static propTypes = {
     tab: PropTypes.string.isRequired,
     user: PropTypes.object.isRequired,
     setTab: PropTypes.func.isRequired,
     updatePassword: PropTypes.func.isRequired,
   };
 
+  onUserSettingsSaved(details) {
+    this.setState({
+      updateUserResult: {
+        success: true,
+        data: { message: t`Account updated successfully!` },
+      },
+    });
+  }
+
+  onUserSettingsSaveFailed(errors) {
```

Comment:
Also, shouldn't we be surfacing the actual errors in the message?

Reply:
Displaying the actual error message was already supported (by EntityForm ect)

This PR added the 'Account Update Failed!' part:

![image](https://user-images.githubusercontent.com/674285/74386660-67890200-4dab-11ea-9852-6e89db37308e.png)